/pybullet_ros:
  ros__parameters:
    plugins:
    - pybullet_ros.plugins.control:Control
    - pybullet_ros.plugins.joint_state_pub:JointStatePub
    - pybullet_ros.plugins.odometry:SimpleOdometry
    - pybullet_ros.plugins.rgbd_camera:RGBDCamera
    
    loop_rate:              120.0           # default: 80.0, the frequency at which to step sim in hz
    gravity:                -9.81           # default: -9.81, earth gravity in m/s^2
    max_effort:             100.0           # default: 50.0, the max force to apply to the model in vel mode
    # True = allow pybullet to compute the inertia tensor based on mass and volume of the collision shape
    # False = you can provide a more accurate inertia tensor, so inertia will be read then from URDF model
    use_inertia_from_file: False           # default: False

/pybullet_ros_control:
  ros__parameters:
    loop_rate:              100.0
    max_effort:             100.0

/pybullet_ros_joint_state_pub:
  ros__parameters:
    loop_rate:              100.0

/pybullet_ros_odometry:
  ros__parameters:
    loop_rate:              60.0

/pybullet_ros_rgbd_camera:
  ros__parameters:
    loop_rate:              30.0
    image_mode:             depth
    rgbd_camera/frame_id:   camera_link
    rgbd_camera/resolution/encoding: 32FC1
    rgbd_camera/resolution/width:  1280
    rgbd_camera/resolution/height: 720
    rgbd_camera/resolution/step: 5120
    rgbd_camera/hfov: 110.0
    rgbd_camera/vfov: 70.0
    rgbd_camera/near_plane: 1.0
    rgbd_camera/far_plane: 20.0
    rgbd_camera/resolution/is_bigendian: False